### PR TITLE
Add AzureOpenAI in docs

### PR DESCRIPTION
**Pull request by akihikokuroda**
### Which issue(s) does this pull-request address?

This was missing in #201 
https://github.com/i-am-bee/bee-agent-framework/pull/201#pullrequestreview-2475048765

### Checklist

- [x] I have read the [contributor guide](https://github.com/i-am-bee/bee-agent-framework/blob/main/CONTRIBUTING.md)
- [x] Linting passes: `yarn lint` or `yarn lint:fix`
- [x] Formatting is applied: `yarn format` or `yarn format:fix`
- [x] Unit tests pass: `yarn test:unit`
- [x] E2E tests pass: `yarn test:e2e`
- [ ] Tests are included 
- [ ] Documentation is changed or added
- [x] Commit messages and PR title follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: docs/llms.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ To unify differences between various APIs, the framework defines a common interf
 | ------------------------------------------------------------------------- | -------------------------- | --------------------------------------------- | ---------------------------------------- |
 | `WatsonX`                                                                 | ✅                         | ⚠️ (model specific template must be provided) | ❌                                       |
 | `Ollama`                                                                  | ✅                         | ✅                                            | ⚠️ (JSON only)                           |
-| `OpenAI`                                                                  | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
+| `OpenAI / Asure OpenAI`                                                   | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
```
Comment (JanPokorny) [inline docs/llms.md:20]: It's "Azure", and it would probably make more sense on a separate line given that it's a separate service

Comment (akihikokuroda) [reply docs/llms.md:20]: It shares the same adapter(same OpenAIChatLLM class).  Only the differences are some env variables and a flag (azure: true) in the initialization.  I can make a separate line if it's better.  WDYT?

Comment (JanPokorny) [reply docs/llms.md:20]: I'd have it as a separate line, it's still technically a different provider

File: docs/llms.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ To unify differences between various APIs, the framework defines a common interf
 | `WatsonX`                                                                 | ✅                         | ⚠️ (model specific template must be provided) | ❌                                       |
 | `Ollama`                                                                  | ✅                         | ✅                                            | ⚠️ (JSON only)                           |
 | `OpenAI`                                                                  | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
+| `Asure OpenAI`                                                            | ❌                         | ✅                                            | ⚠️ (JSON schema only)                    |
```
Comment (JanPokorny) [inline docs/llms.md:21]: Fix the typo please

Comment (akihikokuroda) [reply docs/llms.md]: Thanks!

**Review by Tomas2D — APPROVED** [review]

**Pull request merged**